src/proto/grpc/binary_log/v1alpha/log.proto

```diff
@@ -29,20 +29,20 @@
 
```

Comment:
@kpayson64   if this PR gets delayed, can we at least create a separate PR to fix the log.proto? (it seems to be an independent change).